TruckMuncher/TruckDetailView.swift

```diff
@@ -97,7 +117,7 @@ class TruckDetailView: UIView, UITableViewDataSource, UITableViewDelegate {
     
     // MARK - UITableViewDelegate and UITableViewDataSource Methods
     func numberOfSectionsInTableView(tableView: UITableView) -> Int {
-        return Int(menu!.categories.count)
+        return menu != nil ? Int(menu!.categories.count) : 0
```

Comment:
This is what I choose to swiftify for this PR :smile: `return menu?.categories.count ?? 0`